core/src/main/resources/hudson/model/Messages_es.properties

```diff
@@ -268,35 +270,35 @@ Run.ArtifactsPermission.Description=\
  Este permiso sirve para poder utilizar los artefactos producidos en los proyectos. 
 AbstractProject.AssignedLabelString.InvalidBooleanExpression=Expresi\u00f3n booleana incorrecta: {0}
 ManageJenkinsAction.DisplayName=Administrar Jenkins
-Computer.NoSuchSlaveExists=El nodo {0} no existe. �Quiz�s se est� refiriendo a {1}?
-ResultTrend.StillFailing=Todav�a falla
-Computer.DisconnectPermission.Description=Este permiso permite que los usuarios puedan desconectar nodos o marcarlos como fuera de l�nea.
+Computer.NoSuchSlaveExists=El nodo {0} no existe. \u00BFQuiz\u00E1s se est\u00E1 refiriendo a {1}?
```

Comment:
any reason you changed all these characters?

Reply:
They were UTF8 ones and they should be iso-8859-1. But GitHub prints them right because it uses UTF8 as response charset. 

These characters were getting me crazy because even though I was configured properties files to be iso-8859-1 in IntelliJ, it was opening the file as UTF8 because of how IntelliJ works (but it's another topic).